bin/run

```diff
@@ -25,6 +25,30 @@ wait_for() {
   done
 }
 
+init_connections() {
```

Comment:
why are we adding this? How does this behave if the connections already exist on wtmo?

Reply:
Check out the commit, it has a rationale for doing it this way.

Reply:
Setting the connections in docker-compose guarantees that we do not introduce it into our prod deployment. We also want to see errors if these connections are not set in prod, rather than guessing if they have been set in the UI correctly.

Reply:
I've confirmed that this does _not_ overwrite existing connections.

As it is, we're guessing now if they have been set correctly - we don't know until the job runs. All we can check is that they are set, so this doesn't change that behavior. Thoughts?

Reply:
I don't see a need to modify our existing prod deployment. This introduces a new set of potential failure cases

Reply:
Understood. I've updated to just set these in development.